scripts/windows-install-builder.nsi

```diff
@@ -24,7 +24,8 @@ Name "InaSAFE [[VERSION]]"
 OutFile "InaSAFE-[[VERSION]]-plugin.exe"
 
 # define installation directory
-InstallDir $DOCUMENTS\..\.qgis2\python\plugins\inasafe
+# TODO(IS): currently only install to default profile directory.
+InstallDir $DOCUMENTS\..\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\inasafe
```

Comment:
Users might not have the `default` profile folder (low probability). Is-it possible to check which folder exists in `AppData\Roaming\QGIS\QGIS3\profiles\` and ask in which folder to install?

Reply:
There must be. But it seems the time doesn't allow me to spend too long in this win installer.
I will assume that this installer is being used for beginner user (or intermediate), and most probably still have `default` profile and use it.

Non-beginner user will be able to install InaSAFE easily :)